owlbot.py

```diff
@@ -56,6 +56,7 @@
         # Multi-processing note isn't relevant, as bigquery-magics is responsible for
         # creating clients, not the end user.
         "docs/multiprocessing.rst",
+        "noxfile.py",
```

Comment:
Please don't modify the noxfile in this way. There is a feature in the template that does what we want.

https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/noxfile.py.j2#L75C1-L75C27

Use unit_test_extras_by_python to make sure we have some Python versions where we test without extras and some where we test with.

Reply:
Change reverted